Book.java

```diff
@@ -0,0 +1,38 @@
+import java.util.ArrayList;
```

Comment:
package project.model;
This will have to be added to each file, and create a src/model folder for them to go into. Or you can pull down the main branch and create a new branch and add these files.